package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Production server for running Ember applications using FastBoot",
   "main": "./lib/server",
   "scripts": {
+    "postinstall": "scripts/is-not-legacy-vm || npm install contextify@^0.1.11",
```

Comment:
@ronco The legacy VM behavior is only necessary on 0.10, so what you're describing sounds correct. Good catch on the lingering `contextify` dep in `package.json`; I've removed it.